frontend/lib/src/components/elements/Metric/Metric.test.tsx

```diff
@@ -68,25 +69,26 @@ describe("Metric element", () => {
         value: LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED,
       },
     })
-    const wrapper = mount(<Metric {...props} />)
-    expect(wrapper.find("StyledMetricLabelText").prop("visibility")).toEqual(
-      LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED
+    render(<Metric {...props} />)
+    expect(screen.getByTestId("stMetricLabel")).toHaveAttribute(
+      "visibility",
+      String(LabelVisibilityMessageProto.LabelVisibilityOptions.COLLAPSED)
     )
   })
 
   it("renders direction icon based on props - red/up", () => {
     const props = getProps()
-    const wrapper = mount(<Metric {...props} />)
-    expect(wrapper.find("StyledMetricDeltaText").find("svg")).toBeDefined()
+    render(<Metric {...props} />)
+    expect(screen.getByTestId("stMetricDeltaIcon")).toBeInTheDocument()
```

Comment:
Running into that issue with testing specific `emotion-icons` 😓 
I would add the below to check that it renders in red
```
expect(screen.getByTestId("stMetricDelta")).toHaveStyle("color: #ff2b2b")
```

Reply:
Nvm guess we check the colors in a test below - the only way I can think to test this accurately is passing the direction (up/down) in the `testid` prop passed to `Icon` component -> so when arrowUpward used,
`testid="stMetricDeltaIcon-up"`